eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -42,6 +42,13 @@ public bool SetupEnvironment()
                 var dotnetEFFullPath = Path.Combine(nugetRestore, helixDir, "dotnet-ef.exe");
                 Console.WriteLine($"Set DotNetEfFullPath: {dotnetEFFullPath}");
                 EnvironmentVariables.Add("DotNetEfFullPath", dotnetEFFullPath);
+                var appRuntimePath = $"{Options.DotnetRoot}/shared/Microsoft.AspNetCore.App/{Options.RuntimeVersion}";
+                Console.WriteLine($"Set ASPNET_RUNTIME_PATH: {appRuntimePath}");
+                EnvironmentVariables.Add("ASPNET_RUNTIME_PATH", appRuntimePath);
+                
+                var appRefPath = $"{Options.DotnetRoot}/packs/Microsoft.AspNetCore.App.Ref/{Options.RuntimeVersion}";
```

Comment:
This path won't exist unless the targeting packs are building. Should check for existence before setting the environment variable. Tests will otherwise fail after RTM.

Reply:
I'll just update the tests to check for the paths to exist and no-op if they don't

Reply:
Actually looks like we don't need this variable anyways